mandatory/2-function-creation.js

```diff
@@ -25,7 +39,12 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+
+function tidyUpStrings(arrayOfStrings) {
+  let trim = arrayOfStrings.trim();
+  let result = trim.replaceAll(`/`, ``).toLowercase();
```

Comment:
This is an array so you will need loop in here, I know you did not know about arrays yet, but now you know so try using for loop to solve this, if you still struggling then give me a shout so we can help. Thanks